### ISSUE TITLE
Some cities at low zoom are spelled incorrectly

### ISSUE THREAD

**Issue by nvkelso** [tilezen/vector-datasource#1140]
* **What did you see?** Cities like Zurich and Montreal are spelled weird, and are missing their diacritical marks.
* **What did you expect to see?** Correct spellings.

**Comment by nvkelso** [comment]
For instance:

<img width="341" alt="screen shot 2016-12-07 at 16 11 14" src="https://cloud.githubusercontent.com/assets/853051/20992273/cd9394f0-bc97-11e6-88d8-9c08f1dc2b3a.png">

<img width="288" alt="screen shot 2016-12-07 at 16 11 05" src="https://cloud.githubusercontent.com/assets/853051/20992274/cdac35d2-bc97-11e6-8277-3b063cae1749.png">

**Comment by nvkelso** [comment]
This was an upstream data problem in Natural Earth (character encoding conversion bug). It's been fixed in https://github.com/nvkelso/natural-earth-vector/issues/172... but we're waiting on a new release of Natural Earth before we can take an update, then do a data package onto S3, and then a new production build.

**Comment by freeze2000f** [comment]
Thanks. What could one do to work around the problem of incorrect spelling in the meantime? Is it possible to display osm-data labels at lower zoom levels using the _mapzen vector tile service_? Using a local copy of  _naturalearth populated places dataset_ (with corrected spelling) didn't work for me - but this is another issue.

**Comment by hjanetzek** [comment]
I see you are on it - People from Münster will appreciate :)
![m-0328-1-2017-03-29-160304](https://cloud.githubusercontent.com/assets/1747/24458585/9d44f780-1499-11e7-9e9f-b46f33b5e1e1.png)

**Comment by nvkelso** [comment]
Let's do this with a data migration for this release while Natural Earth (ahem) sort's out it's funk.

@zerebubuth here's the names that need to change:

GEONAMEID | NAME_new | NAME_old | ADM0NAME | LATITUDE | LONGITUDE
--------- | -------- | -------- | -------- | -------- | ---------
255274 | Piraiévs | Piraiavs | Greece | 37.9500207713 | 23.6999897855
380174 | Ad Damazīn | Ad Damazin | Sudan | 11.7704042755 | 34.3499857165
3094802 | Kraków | Kraknw | Poland | 50.0619251261 | 19.9580654926
6325494 | Québec | Qumbec | Canada | 46.8399690949 | -71.2456101905
3135020 | Svolvær | Svolvar | Norway | 68.2332885937 | 14.5666971422
6324733 | St. John’s | St. Johncs | Canada | 47.5849882193 | -52.6810069189
3160881 | Bodø | Bodi | Norway | 67.2467722687 | 14.3990113234
3355672 | Lüderitz | Ldderitz | Namibia | -26.6480000623 | 15.1594258206
6354943 | Egilsstaðir | Egilsstadir | Iceland | 65.2674231152 | -14.3949976008
2627309 | Sauðárkrókur | Sauulrkrdkur | Iceland | 65.7464119661 | -19.6389927587
3444924 | Vitória | Vitiria | Brazil | -20.3220474491 | -40.3679421981
3093133 | Lódz | Lddz | Poland | 51.7769367228 | 19.4494143695
3388368 | São Luís | Sdo Lu1s | Brazil | -2.5140388227 | -44.2679367049
3133895 | Tromsø | Troms- | Norway | 69.635076227 | 18.9920252458
3355624 | Maltahöhe | Maltah1he | Namibia | -24.8399967257 | 16.9399889716
2692969 | Malmö | Malmn | Sweden | 55.5833372183 | 13.0333023668
358448 | Damanhûr | Damanh1r | Egypt | 31.0504419139 | 30.4700158272
3860259 | Córdoba | Cdrdoba | Argentina | -31.3980122115 | -64.1842404159
3110044 | San Sebastián | San Sebastiln | Spain | 43.3203906444 | -1.9799931248
2624652 | Århus | Prhus | Denmark | 56.1572039989 | 10.2106839586
2711537 | Göteborg | Gdteborg | Sweden | 57.7500008333 | 12.0000321031
2657896 | Zürich | Zdrich | Switzerland | 47.381933671 | 8.5480642718
777073 | Vadsø | Vadsl | Norway | 70.0965967757 | 29.7657492666
6169141 | Trois-Rivières | Trois-Rivipres | Canada | 46.3499731639 | -72.5499491798
2702979 | Jönköping | Jdnknping | Sweden | 57.7713431998 | 14.165016234
2720383 | Borlänge | Borllnge | Sweden | 60.4833223665 | 15.4166711005
2686469 | Örnsköldsvik | Prnskaldsvik | Sweden | 63.3180072216 | 18.7166763903
2664454 | Västerås | Vdster1s | Sweden | 59.6300152782 | 16.5400133857
2618425 | København | Kdbenhavn | Denmark | 55.680510049 | 12.5615398887
3415667 | Ísafjörður | Psafjareur | Iceland | 66.0833164664 | -23.1499670832
2516395 | Jaén | Japn | Spain | 37.7703934927 | -3.7999853937
3415496 | Keflavík | Keflavtk | Iceland | 64.016646748 | -22.5665918391
2720679 | Bollnäs | Bollnas | Sweden | 61.3520031933 | 16.3665872789
2712414 | Gävle | Gdvle | Sweden | 60.6669804068 | 17.1666418037
636464 | Sodankylä | Sodankyld | Finland | 67.4170593457 | 26.6000194893
2663536 | Växjö | Vdxjn | Sweden | 56.8836971223 | 14.8167077216
3155573 | Gjøvik | Gjpvik | Norway | 60.8000213818 | 10.700008096
655194 | Jyväskylä | Jyviskylc | Finland | 62.2603456819 | 25.7499938545
5880054 | Utqiaġvik | Barrow | United States of America | 71.290569681 | -156.7885799581
2686657 | Örebro | Arebro | Sweden | 59.2803466991 | 15.2199905992
3141310 | Rørvik | Rorvik | Norway | 64.86801597 | 11.2053002509
2688368 | Norrköping | Norrkaping | Sweden | 58.5954272656 | 16.1786917711
6077243 | Montréal | Montraal | Canada | 45.5019450642 | -73.5852428167
4013708 | Ciudad Juárez | Ciudad Ju rez | Mexico | 31.6923228718 | -106.4919939155
602913 | Skellefteå | Skellefte | Sweden | 64.7720786735 | 20.9500284411
596128 | Panevežys | Panevetys | Lithuania | 55.740020161 | 24.3700264066
2867543 | Münster | Monster | Germany | 51.9704052929 | 7.6200410549
2667303 | Trollhättan | Trollhtttan | Sweden | 58.267101053 | 12.2999621161
2934246 | Düsseldorf | Ddsseldorf | Germany | 51.2203735546 | 6.7799889716
2720501 | Borås | Borus | Sweden | 57.7304413036 | 12.9199759508
2861650 | Nürnberg | Nornberg | Germany | 49.4499906607 | 11.0799849026
3119841 | La Coruña | La Coruea | Spain | 43.3299766226 | -8.4199876317
2856883 | Osnabrück | Osnabrtck | Germany | 52.2804380484 | 8.0499889716
3413829 | Reykjavík | Reykjav k | Iceland | 64.1500236197 | -21.9500144872
3134331 | Tønsberg | Tdnsberg | Norway | 59.2640010895 | 10.4210014708
2694762 | Linköping | Linknping | Sweden | 58.4100122265 | 15.6299397366
2685750 | Östersund | Astersund | Sweden | 63.1833126009 | 14.6499995511
602150 | Umeå | Umei | Sweden | 63.8299914746 | 20.2399942614
3836873 | Tucumán | Tucum-n | Argentina | -26.814054285 | -65.2185700465
604490 | Luleå | Lulen | Sweden | 65.5966347723 | 22.1583783294
3396016 | Macapá | Macap- | Brazil | 0.0330070179 | -51.0500212012

**Comment by zerebubuth** [comment]
That first one, Piraiévs, looks a bit odd to me. I think it should be [Piraeus](https://en.wikipedia.org/wiki/Piraeus) instead.

**Comment by nvkelso** [comment]
Following up about `Piraiévs`, I see that Geonames.gov has the `v` spelling, but other places have the `u`. 

Let's use `v` now as in the PR and I'll followup with fix in Natural Earth in https://github.com/nvkelso/natural-earth-vector/issues/194.

**Comment by nvkelso** [comment]
Verified on dev:

<img width="260" alt="screen shot 2017-04-28 at 11 57 37" src="https://cloud.githubusercontent.com/assets/853051/25543068/eb6f142c-2c09-11e7-8a9e-d216a61d4715.png">
<img width="397" alt="screen shot 2017-04-28 at 11 57 18" src="https://cloud.githubusercontent.com/assets/853051/25543069/eb75d424-2c09-11e7-941f-cf3fb691599f.png">
